### PR TITLE
merge mysql v2 into v1

### PR DESCRIPTION
**Pull request by rodireich**
## What
Progressive rollout to source-mysql V2

### CODE REVIEW

**Comment by rodireich** [comment]
@clnoll I would like to try to merge this to master to test progressive rollout.
I merged the code of source-mysql-v2 into source-mysql, so this is effectively a non breaking version of source-mysql.
It doesn't like something about the version.

File: airbyte-integrations/connectors/source-mysql/metadata.yaml
Comment (rodireich) [inline airbyte-integrations/connectors/source-mysql/metadata.yaml:1]: @clnoll I wanted to make sure that once merged to master, this won't upgrade all existing cloud connections.

File: .github/workflows/connectors_version_increment_check.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
     name: Connectors Version Increment Check
     runs-on: connector-test-large
     if: github.event.pull_request.head.repo.fork != true
-    timeout-minutes: 12
+    timeout-minutes: 22
```
Comment (rodireich) [inline .github/workflows/connectors_version_increment_check.yml:26]: This was timing out

File: airbyte-integrations/connectors/source-mysql/metadata.yaml [hunk 2]
```diff
@@ -22,54 +22,11 @@ data:
       enabled: true
     oss:
       enabled: true
-  releaseStage: generally_available
-  releases:
-    breakingChanges:
-      3.0.0:
-        message: Add default cursor for cdc
-        upgradeDeadline: "2023-08-17"
+  releaseStage: alpha
```
Comment (rodireich) [inline airbyte-integrations/connectors/source-mysql/metadata.yaml:25]: @clnoll Should this return to alpha? I wasn't sure as source-mysql was already generally_available

**Review by xiaohansong — APPROVED** [review]

File: airbyte-integrations/connectors/source-mysql/metadata.yaml (hunk 2 above)
Comment (clnoll) [reply airbyte-integrations/connectors/source-mysql/metadata.yaml:25]: I'm not sure what the `releaseStage` refers to - it's not related to progressive rollouts. Maybe @alafanechere knows?

**Review by clnoll — APPROVED** [review]
The progressive rollout updates in metadata.yaml look good to me!

File: airbyte-integrations/connectors/source-mysql/metadata.yaml (hunk 2 above)
Comment (natikgadzhi) [reply airbyte-integrations/connectors/source-mysql/metadata.yaml:25]: Oh this is my favorite genre! /cc @nataliekwong and @alex-gron. TLDR I think is just keep it what it was :D

Comment (natikgadzhi) [reply airbyte-integrations/connectors/source-mysql/metadata.yaml:25]: I think it only affects internal reporting, not actual product behavior

Comment (nataliekwong) [reply airbyte-integrations/connectors/source-mysql/metadata.yaml:25]: I'm not sure actually what this affects, sorry I can't be of more help!

**Pull request merged**